lib/cylc/cfgspec/suite.py

```diff
@@ -464,33 +464,32 @@ def upg(cfg, descr):
         ['scheduling', 'special tasks', 'external-triggered'],
         ['scheduling', 'special tasks', 'external-trigger'],
     )
-    for key in SPEC['cylc']['events']:
-        u.deprecate(
-            '6.11.0', ['cylc', 'event hooks', key], ['cylc', 'events', key])
-    u.deprecate('6.11.0', ['cylc', 'event hooks'])
-    for key in SPEC['runtime']['__MANY__']['events']:
-        u.deprecate(
-            '6.11.0',
-            ['runtime', '__MANY__', 'event hooks', key],
-            ['runtime', '__MANY__', 'events', key])
-    u.deprecate('6.11.0', ['runtime', '__MANY__', 'event hooks'])
+    u.deprecate(
+        '6.11.0', ['cylc', 'event hooks'], ['cylc', 'events'])
+    u.deprecate(
+        '6.11.0',
+        ['runtime', '__MANY__', 'event hooks'],
+        ['runtime', '__MANY__', 'events'])
+    u.deprecate(
+        '6.11.0',
+        ['runtime', '__MANY__', 'job submission'],
+        ['runtime', '__MANY__', 'job'])
     u.deprecate(
         '6.11.0',
-        ['runtime', '__MANY__', 'job submission', 'method'],
+        ['runtime', '__MANY__', 'job', 'method'],
         ['runtime', '__MANY__', 'job', 'batch system'])
     u.deprecate(
         '6.11.0',
-        ['runtime', '__MANY__', 'job submission', 'command template'],
+        ['runtime', '__MANY__', 'job', 'command template'],
         ['runtime', '__MANY__', 'job', 'batch submit command template'])
     u.deprecate(
         '6.11.0',
-        ['runtime', '__MANY__', 'job submission', 'shell'],
+        ['runtime', '__MANY__', 'job', 'shell'],
         ['runtime', '__MANY__', 'job', 'shell'])
```

Comment:
Well spotted! I've updated the back-port and have put up a new pull (#2103) for 7.0.x.